lib/diagnostic.js

```diff
@@ -29,8 +33,13 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
+const any = function (workingArray, callback) {
 
+    if(callback(element, index, array)) {
+      return true;
+    }
+    return false;
+});
```

Comment:
This line and the next one break the entire script.  It should just be a single closing brace.